benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -74,6 +103,9 @@ def time_and_bool(self):
     def time_or_bool(self):
         (self.b | self.b)
 
+    def time_and_bool_small(self):
+        (self.b_small & self.b_small)
+
```

Comment:
Could this be added as a parameterized set of shapes for all the tests in this class?

Reply:
nevermind, this is good as-is.